README.md

```diff
@@ -142,7 +162,16 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is an outline of what kind of data your table(s) will contain.
+
+If we were using ActiveRecord:
+
+class People < ActiveRecord::Base
```

Comment:
Yes, but to be more in line with convention, it would probably be a class of Person (singular)